smoke-tests/designer/features/steps/steps.js

```diff
@@ -370,7 +370,9 @@ When("I create a section titled {string}", function (sectionTitle) {
   editSection.sectionTitle.setValue(this.sectionTitle);
   editSection.sectionSaveBtn.click();
   editPage.saveBtn.click();
-  expect(formDesigner.pageSectionName(this.pageName)).toHaveText(sectionTitle);
+  browser.waitUntil(
+    () => formDesigner.pageSectionName(this.pageName).getText() === sectionTitle
```

Comment:
will this actually throw an error? does it need to be wrapped in expect? `expect(formDesigner.pageSection...`

Reply:
It does cause a failure if the condition isn't met, however for clarity I've added a custom timeout message in the event it fails. I've purposedly caused the test to fail to show the output below:

<img width="909" alt="Screenshot 2021-04-27 at 13 38 28" src="https://user-images.githubusercontent.com/4447278/116242567-f0484f80-a75d-11eb-9aa8-9254a9f494c8.png">